.github/workflows/diff.yml

```diff
@@ -69,6 +69,13 @@ jobs:
           trigger: '#gogoeditdiff'
         env:
           GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+      - uses: xt0rted/pull-request-comment-branch@v2
+        id: comment-branch
```

Comment:
What does this step exactly do? The below of course makes sense but I am not clear if this is actually making a comment or not

Reply:
it's creating the id to be used as a reference in the next step